expressEngine/customer/service/customer.js

```diff
@@ -9,27 +9,29 @@ import { error, success } from '../../cms/customer';
 
 class Service extends BaseService {
     register = async (data) => {
-        try {
-            const { firstName, lastName, city, country, contactNo } = data;
-            //const requiredFields = ["firstName", "lastName", "city", "country", "contactNo"];
+      try {
+        const {
+          name, email, city, country, contactNo,
+        } = data;
+        // const requiredFields = ["firstName", "lastName", "city", "country", "contactNo"];
 
-            const isExist = await DBService.findOne(Customer, { firstName, lastName });
-            if (isExist) {
-                return error.alreadyRegistered;
-            }
+        const isExist = await DBService.findOne(Customer, { email });
+        if (isExist) {
+          return error.alreadyRegistered;
+        }
 
-            const customer = await DBService.create(Customer, {
-                firstName,
-                lastName,
-                city,
-                country,
-                contactNo,
-            });
+        const customer = await DBService.create(Customer, {
+          name,
+          email,
+          city,
+          country,
+          contactNo,
+        });
 
-            return this.success(customer, success.customerRegistered);
-        } catch (err) {
-            return this.error(err);
-        }
+        return this.success(customer, success.customerRegistered);
```

Comment:
@ayushj117 please check the error and success handling in user, and maintain the same structure for error and success in other files.

Reply:
done